mlir/test/Target/LLVMIR/Import/tune-cpu.ll

```diff
@@ -0,0 +1,9 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
+
+; CHECK-LABEL: llvm.func @tune_cpu()
+; CHECK-SAME: tune_cpu = "pentium4"
```

Comment:
Can you add a check that ensures that this is not also placed in the passthrough dictionary? I don't see any part in the code that would avoid this. Adding it to the `ExplicitAttributes` set in `ModuleImport.cpp` should be enough.

Reply:
From what I can see, L1685 of ModuleImport.cpp has tune-cpu listed among the ExplicitAttributes.  Is this sufficient?  Or is further checking needed?  Thank you so much.

Reply:
Given that we are both not entirely sure what happens, it might be sensible to check for that, just to be sure 🙂

Reply:
Alright, after some digging, I can confirm that `tune-cpu` is listed among the `kExplicitAttributes` and that these attributes are expressly excluded from being added to the passthrough dictionary based on the code in `processPassthroughAttrs` in `ModuleImport.cpp`.